src/main/kotlin/uk/gov/gdx/datashare/controllers/EventsController.kt

```diff
@@ -49,43 +49,9 @@ class EventsController(
   @Operation(
     operationId = "getEvents",
     summary = "Event Get API - Get event data",
-    description = "Get all events for Acquirer, Need scope of events/consume. " +
-      "     This field is only populated when the acquirer has <em>enrichmentFieldsIncludedInPoll</em> enabled, otherwise an empty object." +
-      "     Full dataset for the event can be obtained by calling <pre>/events/{id}</pre>" +
-      "     <h3>Event Types</h3>" +
-      "     <h4>1. Death Notification - Type: <em>DEATH_NOTIFICATION</em></h4>" +
-      "     <p>Death notifications take the following json structure." +
-      "     <pre>\n" +
-      "{\n" +
-      "        \"registrationDate\": \"2023-01-23\",\n" +
-      "        \"firstNames\": \"Mary Jane\",\n" +
-      "        \"lastName\": \"Smith\",\n" +
-      "        \"maidenName\": \"Jones\",\n" +
-      "        \"sex\": \"Male\",\n" +
-      "        \"dateOfDeath\": \"2023-01-02\",\n" +
-      "        \"dateOfBirth\": \"1972-02-20\",\n" +
-      "        \"birthPlace\": \"56 Test Address, B Town\",\n" +
-      "        \"deathPlace\": \"Hospital Ward 5, C Town\",\n" +
-      "        \"occupation\": \"Doctor\",\n" +
-      "        \"retired\": true,\n" +
-      "        \"address\": \"101 Address Street, A Town, Postcode\"\n" +
-      "}\n" +
-      "      </pre>" +
-      "      <p><b>Mandatory Fields</b>: registrationDate, firstNames, lastName, sex, dateOfDeath</p>" +
-      "      <p><b>Gender Types:</b>: Male, Female, Indeterminate</p>" +
-      "<h4>2. Person has been sent to prison - Type: <em>ENTERED_PRISON</em></h4>" +
-      "     <p>Prisoner received notifications take the following json structure." +
-      "     <pre>\n" +
-      "{\n" +
-      "        \"prisonerNumber\": \"A1234DB\",\n" +
-      "        \"firstName\": \"Mary\",\n" +
-      "        \"middleNames\": \"Jane\",\n" +
-      "        \"lastName\": \"Smith\",\n" +
-      "        \"gender\": \"Male\",\n" +
-      "        \"dateOfBirth\": \"1972-02-20\",\n" +
-      "}\n" +
-      "      </pre>" +
-      "      <p><b>Mandatory Fields</b>: prisonerNumber, firstName, lastName, gender, dateOfBirth</p>",
+    description = "<h3>Event Types</h3>" +
```

Comment:
Do we want to remove the `enrichmentFieldsIncludedInPoll` option?

Reply:
I think so. It looks like a performance nightmare

Reply:
https://govukverify.atlassian.net/browse/GPC-197